homeassistant/components/ozw/websocket_api.py

```diff
@@ -213,7 +215,68 @@ def websocket_get_code_slots(hass, connection, msg):
 )
 def websocket_get_config_parameters(hass, connection, msg):
     """Get a list of configuration parameters for an OZW node instance."""
-    _call_util_function(hass, connection, msg, True, get_config_parameters)
+    try:
+        node = get_node_from_manager(
+            hass.data[DOMAIN][MANAGER], msg[OZW_INSTANCE], msg[NODE_ID]
+        )
+    except NotFoundError as err:
+        connection.send_error(
+            msg[ID],
+            websocket_api.const.ERR_NOT_FOUND,
+            err.args[0],
+        )
+        return
+
+    raw_values = get_config_parameters(node)
```

Comment:
Wrap this and the schema generation in a function that returns the payload (`config_params`). Then we can still use `_call_util_function` I think.